src/emmy/abstract/function.cljc

```diff
@@ -230,9 +220,9 @@
           litfns)))
 
 (u/sci-macro with-literal-functions [litfns & body]
```

Comment:
We definitely want this indentation to match `defmacro`... can you try adding

```
{:clj-kondo/lint-as 'clojure.core/defn}
```

to `emmy.util/sci-macro` to see if vscode picks it up? 

Of course

```
{:clj-kondo/lint-as 'clojure.core/defmacro}
```

makes more sense but I think `defn` might be the only supported option...